src/Mod/Path/Path/Dressup/DogboneII.py

```diff
@@ -184,10 +184,10 @@ def boneTip(self):
         return self.bone.tip()
 
     def boneIDs(self):
-        return list(sorted(self.bones.keys()))
+        return sorted(self.bones)
```

Comment:
This changes the return type of the function. What's the rational for this change?